src/main/resources/application-dev.yml

```diff
@@ -6,15 +6,8 @@
 
 # Most values are inherited from - application.yml - except the following:
 
-spring:
-  security:
-    oauth2:
-      resourceserver:
-        jwt:
-          public-key-location: classpath:local-public-key.pub
-
-  jpa:
-    show-sql: true
+jpa:
```

Comment:
this looks wrong - you've moved `jpa:` up a level to the root whereas it should still be under `spring:`

Reply:
Whoops, thanks for the spot Peter. Changed that.